test/hotspot/jtreg/serviceability/jvmti/thread/GetFrameCount/framecnt01/framecnt01.java

```diff
@@ -79,7 +79,7 @@ public static void main(String args[]) throws Exception {
         }
 
         // this is too fragile, implementation can change at any time.
-        checkFrames(vThread1, false, 15);
+        checkFrames(vThread1, false, 14);
```

Comment:
Is this due to the `@hidden` being added to `Continuation.enter()` and `enter0()`? If so, since both methods are now hidden, why are there not 2 fewer frames? Was there also an additional frame added somewhere?

Reply:
No, it's that a lambda expression is replaced in the implementation.